src/Sentry.Maui/Sentry.Maui.csproj

```diff
@@ -6,11 +6,11 @@
       Target net6.0 so we can run unit tests on platform-neutral code.
       Target other platforms so we can include platform-specific code, and bundle native SDKs.
     -->
-    <TargetFrameworks>net7.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(NO_ANDROID)' == ''">$(TargetFrameworks);net7.0-android</TargetFrameworks>
-    <TargetFrameworks Condition="'$(NO_IOS)' == '' And $([MSBuild]::IsOSPlatform('OSX'))">$(TargetFrameworks);net7.0-ios</TargetFrameworks>
-    <TargetFrameworks Condition="'$(NO_MACCATALYST)' == '' And $([MSBuild]::IsOSPlatform('OSX'))">$(TargetFrameworks);net7.0-maccatalyst</TargetFrameworks>
-    <TargetFrameworks Condition="'$(NO_WINDOWS)' == '' And !$([MSBuild]::IsOSPlatform('Linux'))">$(TargetFrameworks);net7.0-windows10.0.19041.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;net7.0</TargetFrameworks>
```

Comment:
Do we have it in testing already? We need to cover both builds now

Reply:
Sentry.Maui.Tests targets 7 and 8, yes 👍🏻